src/BizHawk.Emulation.Cores/Consoles/Nintendo/Gameboy/Gambatte.cs

```diff
@@ -101,7 +101,27 @@ public Gameboy(CoreComm comm, GameInfo game, byte[] file, Gameboy.GambatteSettin
 
 				if (_syncSettings.EnableBIOS)
 				{
-					bios = comm.CoreFileProvider.GetFirmware(biosSystemId, biosId, true, "BIOS Not Found, Cannot Load.  Change SyncSettings to run without BIOS.");
+					try
+					{
+						bios = comm.CoreFileProvider.GetFirmware(biosSystemId, biosId, true, "BIOS Not Found, Cannot Load.  Change SyncSettings to run without BIOS.");

```

Comment:
Use `required: false` here if you're going to have a fallback instead of `catch (MissingFirmwareException)`